README.md

```diff
@@ -20,3 +22,5 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+Starting with generating all permutatiosn, this would take a complexity of n factorial (O(n!)). In the loop that checks each permutation if it is sorted or not, we call checkSort which has a for loop being O(n) This would all result in O(n * n!) being average and worst case. The other for loops not mentioned are not in the time complexity since these override or are constant. The best case input for this implementation would be a sorted array, since it checks it first thing. The worst case would be an array going in descending order since it would try all permutations and the sorted one being last. The worst case will stay at O(n * n!). In some cases, randomly permutating may help time complexities, which has a possibility of improving the average cases, however, this is not guaranteed.
```

Comment:
What's the best case complexity? And is there a guarantee that random permutations without memory would ever find the solution?